source/Octopus.Tentacle.Tests/Commands/RegisterWorkerCommandFixture.cs

```diff
@@ -88,7 +88,7 @@ public void ShouldRegisterListeningWorker()
                 "--force",
                 "--proxy=Proxy");
 
-            Assert.That(operation.WorkerPoolNames.Single(), Is.EqualTo("SomePool"));
+            Assert.That(operation.WorkerPools.Single(), Is.EqualTo("SomePool"));
```

Comment:
@rain-on it would make sense and I think is a requirement that the new k8s command is tested similar to this in this repo. (on top of the E2E test in Octopus Deploy)

Reply:
Added the unit tests.

With regards to the E2E tests in Octopus, I am in the process of updating them to cover the Tentacle changes here (using the branch build). Will share the Teamcity link to those once they are available.